openeo/local/processing.py

```diff
@@ -59,4 +68,74 @@ def load_local_collection(*args, **kwargs):
 PROCESS_REGISTRY["load_collection"] = Process(
     spec=openeo_processes_dask.specs.load_collection, 
     implementation=load_local_collection,
+)
+
+def resample_cube_spatial_rioxarray(*args, **kwargs):
+    pretty_args = {k: type(v) for k, v in kwargs.items()}
+    _log.debug(f"Running process resample_cube_spatial")
+    _log.debug(f"kwargs: {pretty_args}")
+    _log.debug("-" * 80)
+
+    methods_list = {
+        "near": 0,
+        "bilinear": 1,
+        "cubic": 2,
+        "cubicspline": 3,
+        "lanczos": 4,
+        "average": 5,
+        "mode": 6,
+        "gauss": 7,
+        "max": 8,
+        "min": 9,
+        "med": 10,
+        "q1": 11,
+        "q3": 12,
+        "sum": 13,
+        "rms": 14
+    }
+    data = kwargs['data']
+    target = kwargs['target']
+    method = kwargs['method']
```

Comment:
if `data`, `target` and `method` are required: why not defining these arguments directly instead of using `kwargs`? It makes things more clear, self-documenting, you can add type annotations, the error message are nicer, ...

Reply:
Done